src/test/java/io/javalin/TestHttpResponseExceptions.kt

```diff
@@ -47,7 +48,8 @@ class TestHttpResponseExceptions {
         assertThat(response.body, `is`("""{
                 |    "title": "Off limits!",
                 |    "status": 403,
-                |    "type": "https//javalin.io/documentation#ForbiddenResponse"
+                |    "type": "https//javalin.io/documentation#ForbiddenResponse",
```

Comment:
I'm guessing that your test succeeds but there's a missing colon (`:`) there

Reply:
Thanks, fixed.